packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -20,15 +20,36 @@ export default class SSRRenderer extends BaseRenderer {
   }
 
   renderScripts (renderContext) {
-    return renderContext.renderScripts()
+    let scripts = renderContext.renderScripts()
+    const scriptPattern = /<script[^>]*?src="([^"]*?)"[^>]*?>[^<]*?<\/script>/g
+
+    scripts = scripts.replace(scriptPattern, (scriptTag, jsFile) => {
```

Comment:
Can we refactor this functionality (parts repeated in `renderScripts` and `renderResourceHints`)